perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/GroupsManagerBlImpl.java

```diff
@@ -114,12 +117,16 @@ public void deleteMembersGroup(PerunSession sess, Vo vo) throws InternalErrorExc
 		try {
 			group = getGroupByName(sess, vo, VosManager.MEMBERS_GROUP);
 		} catch (GroupNotExistsException e) {
-			throw new ConsistencyErrorException("Built-in members group must exists.",e);
+			throw new ConsistencyErrorException("Built-in members group must exist.",e);
 		}
 		try {
 			this.deleteAnyGroup(sess, group, true);
 		} catch (RelationExistsException e) {
-			throw new ConsistencyErrorException("Built-in members group cannot have any relation in this stage.",e);
+			throw new ConsistencyErrorException("Built-in members group cannot have any relation in this stage.", e);
+		} catch (NotGroupMemberException | WrongAttributeValueException |
+				WrongReferenceAttributeValueException | GroupNotExistsException | AlreadyMemberException e) {
+			throw new ConsistencyErrorException("This exception is caused by something going wrong while deleting groups operations. " +
+					"Members group cannot have any operations defined on it.", e);
 		}
```

Comment:
Better to create new exception - something like "GroupOperationException" and use it in method processRelation for all exceptions there. Then let it throw here.